web/src/main/java/io/spine/web/package-info.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2000-2018 TeamDev Ltd. All rights reserved.
+ * TeamDev PROPRIETARY and CONFIDENTIAL.
+ * Use is subject to license terms.
+ */
+
+/**
+ * This package contains the basic client transport API.
```

Comment:
I don't think this package is only about a transport.

Also, please rephrase `basic`.